packages/moleculer-db-adapter-mongoose/src/index.js

```diff
@@ -383,6 +383,29 @@ class MongooseDbAdapter {
 		return entity;
 	}
 
+	/**
+	 * Convert hex string to ObjectID
+	 *
+	 * @param {String} id
+	 * @returns {ObjectID}
+	 *
+	 * @memberof MongooseDbAdapter
+	 */
+	stringToObjectId (id) {
+    return new mongoose.Schema.Types.ObjectId(id);
```

Comment:
I think that we should check if the `id` is valid. Just as done in mongo adapter
https://github.com/moleculerjs/moleculer-db/blob/81919e05a67ac9c973f4743517bcee7a6013aabf/packages/moleculer-db-adapter-mongo/src/index.js#L360-L365

I think that `isValid` method is avalidable as `mongoose.Types.ObjectId.isValid(id)`